src/app/models/vehicle-test-result-update.ts

```diff
@@ -0,0 +1,7 @@
+import { UserDetails } from '@app/models/user-details';
+import { TestResultModel } from '@app/models/test-result.model';
+
+export interface VehicleTestResultUpdate {
+  testResult: TestResultModel;
+  msUserDetails: UserDetails;
```

Comment:
I guess this is addressed as part of 10282.
Just checked the backend and it seems that it is not expecting this shape,
I believe it is expecting 
```
  ....,
  "testStationName": "",
   "testerEmailAddress": "",
  ....
```
Will have to check with @castancu or @alex-deloitte or @m-ujaffer on that one

Reply:
the PUT call needs the msUserDetails